microshift_install/microshift-fips.adoc

```diff
@@ -16,8 +16,4 @@ include::modules/microshift-fips-rpm-system.adoc[leveloffset=+1]
 [id="additional-resources_microshift-fips_{context}"]
```

Comment:
I don't know if the order matters or not but there is [this](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies:~:text=Example%20unique%20anchor%20ID%20for%20an%20%22Additional%20resources%22%20title) in the contribution guide.